src/server.ts

```diff
@@ -626,7 +630,13 @@ export class Server {
   public async sendTransaction(
     transaction: Transaction | FeeBumpTransaction,
   ): Promise<SorobanRpc.SendTransactionResponse> {
-    return await jsonrpc.post(
+    return this._sendTransaction(transaction).then(parseRawSendTransaction);
```

Comment:
for consideration, a test to demonstrate this new expectation, like `it provides an encoded error result`  in https://github.com/stellar/js-soroban-client/blob/decode-send-transaction/test/unit/server/send_transaction_test.js#L38

Reply:
good call :+1:

Reply:
done in 930ef87!